repo/packages/A/arangodb/0/config.json

```diff
@@ -0,0 +1,146 @@
+{
+  "type": "object",
+  "properties": {
+
+    "mesos": {
+      "description": "Mesos specific configuration properties",
+      "type": "object",
+      "properties": {
+        "master": {
+          "description": "The URL of the Mesos master. The format is a comma-delimited list of hosts like zk://host1:port,host2:port/mesos. If using ZooKeeper, pay particular attention to the leading zk:// and trailing /mesos! If not using ZooKeeper, standard URLs like http://localhost are also acceptable.",
+          "type": "string",
+          "default": "zk://master.mesos:2181/mesos"
+        }
+      },
+      "required": [
+        "master"
+      ]
+    },
+
+    "arangodb": {
+      "description": "ArangoDB specific configuration properties",
+      "type": "object",
+      "properties": {
+
+        "id": {
```

Comment:
this probably seems crazy :) please switch the property id to framework-name.   or add framework-name.   we are standardizing on using just 1 of these properties `framework-name` hence the request to change the name.  Note that there is special magic that ties the health check display on the dcos ui to the framework if 2 things are true:  1) the framework has a property of framework: true (arangodb already does) and 2) there is a property of framework-name with a default value.

Also note... a change of id here requires a change of id under the required section plus changes to all references of id in the marathon.json